WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -339,63 +316,59 @@ final public class PushNotificationsManager: NSObject {
 
         return true
     }
+}
 
 
-
-
-    // MARK: - Private Methods: Helpers
-
+// MARK: - Private Methods
+//
+private extension PushNotificationsManager {
 
     /// Tracks a Notification Event
     ///
     /// - Parameter userInfo: The Notification's Payload
     ///
-    fileprivate func trackNotificationWithUserInfo(_ userInfo: NSDictionary) {
+    func trackNotification(with userInfo: NSDictionary) {
         var properties = [String: String]()
 
-        if let noteId = userInfo.number(forKey: notificationIdentifierKey) {
-            properties[trackingIdentifierKey] = noteId.stringValue
+        if let noteId = userInfo.number(forKey: Constants.notificationIdentifierKey) {
+            properties[Tracking.identifierKey] = noteId.stringValue
         }
 
-        if let type = userInfo.string(forKey: notificationTypeKey) {
-            properties[trackingTypeKey] = type
+        if let type = userInfo.string(forKey: Constants.notificationTypeKey) {
+            properties[Tracking.typeKey] = type
         }
 
         if let theToken = deviceToken {
-            properties[trackingTokenKey] = theToken
+            properties[Tracking.tokenKey] = theToken
         }
 
         let event: WPAnalyticsStat = (applicationState == .background) ? .pushNotificationReceived : .pushNotificationAlertPressed
         WPAnalytics.track(event, withProperties: properties)
     }
+}
 
 
+// MARK: - Nested Types
+//
+private extension PushNotificationsManager {
 
-    /// Parses the NSData sent by Apple's Push Service, and extracts the Device Token
-    ///
-    fileprivate func parseTokenFromAppleData(_ tokenData: Data) -> String {
-        return tokenData.hexString
+    enum Constants {
```

Comment:
I was actually going to say something that's kind of similar to what @nheagy said, but opposite in a certain way.

The name `Constants` is just really generic.

Since we don't store variables inside an `enum Variables`, or methods in an `enum Methods`... can we at least make the name less generic?

The constants inside this enum could be divided into something like `enum Device` and `enum Notification` (or whatever name is descriptive of what they are).